UITests/Responses/Transactions/TransactionsEmptyList.json

```diff
@@ -0,0 +1,27 @@
+{
+    "count": 0,
+    "offset": 0,
+    "limit": 10,
+    "data": [
+    ],
+    "links": [
+        {
+            "params": {
+                "rel": "self"
+            },
+            "href": "https://qamaster-hyperwallet.aws.paylution.net/rest/v3/users/usr-token/receipts?offset=0&limit=10"
+        },
+        {
+            "params": {
```

Comment:
empty list does not have anything in the response. Just response status as 204

Reply:
renamed

Reply:
modified test to receive http code 204